viewer/data/meson.build

```diff
@@ -4,7 +4,7 @@ appdata_conf = configuration_data()
 appdata_conf.set ('ARAVIS_API_VERSION', aravis_api_version)
 
 configure_file (input: files('org.aravis.viewer.appdata.xml.in'),
-		output: 'org.aravis.viewer-@0@.appdata.xml'.format (aravis_api_version),
+		output: 'org.aravis.viewer.metainfo.xml'.format (aravis_api_version),
```

Comment:
Different aravis series are supposed to be installable in parallel. This change breaks this possibility.

Reply:
Understood!
The last commit should have fixed this issue, right?

Reply:
nope... I have to modify the id tag too... :) coming soon...

Reply:
Not completely. In the bug report you mentioned, it is said the filename does not patch the component ID. I guess the `<id>` node should be modified, like what is done for the `<launchable>` node.